lib/logstash/environment.rb

```diff
@@ -76,10 +76,6 @@ def jruby?
       @jruby ||= !!(RUBY_PLATFORM == "java")
     end
 
-    def windows?
-      Gem.win_platform?
-    end
```

Comment:
This is need here because you requested to have automatic defaults per platform the complete environment is not loaded for the plugin manager.